src/ngx_http_lua_kong_module.h

```diff
@@ -36,5 +36,8 @@ ngx_flag_t
 ngx_http_lua_kong_get_upstream_ssl_verify(ngx_http_request_t *r,
     ngx_flag_t proxy_ssl_verify);
 
+ngx_flag_t
+ngx_http_lua_kong_ssl_get_http2_alpn_enabled(ngx_ssl_connection_t *ssl,
```

Comment:
this function seems not used?

Reply:
https://github.com/Kong/kong/pull/13709/files#diff-4242fc0c9bdeb3525f6463c1cec41f726e360b0b8b897be3abab6d83863c649fR21 used in kong side openresty patch